pkg/manager/member/pd_member_manager.go

```diff
@@ -741,17 +741,7 @@ func getPDConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 		config.Dashboard.TiDBCertPath = path.Join(tidbClientCertPath, corev1.TLSCertKey)
 		config.Dashboard.TiDBKeyPath = path.Join(tidbClientCertPath, corev1.TLSPrivateKeyKey)
 	}
-	// TiFlash requires PD to enable the `replication.enable-placement-rules`
-	// Check detail in https://pingcap.com/docs/stable/reference/tiflash/deploy/
-	if tc.Spec.TiFlash != nil {
```

Comment:
Enable placement rules if we create the cluster?

Reply:
When creating the configmap, it does not know whether the cluster is new or not, and it should not care about it.
I think we can make it clear in the doc, after all, users also have to add the TiFlash configurations, the TiFlash docs also request users to enable it manually.

Reply:
Currently there are 2 steps for user to enable Tiflash. First, enable rule-placement by pd-ctl. Second configure Tiflash Table Replicas by sql. I think only SQL configuration should be exposed to the users excluding pdapi. We have some seniors that users couldn't get in touch with pdapi. 

As TiFlash is [supprted to be enabled in an existed cluster](https://pingcap.com/docs-cn/stable/reference/tiflash/deploy/#%E5%9C%A8%E5%8E%9F%E6%9C%89-tidb-%E9%9B%86%E7%BE%A4%E4%B8%8A%E6%96%B0%E5%A2%9E-tiflash-%E7%BB%84%E4%BB%B6), I think we could enabled it `placement rule` by pdapi before syncing tiflash.

Reply:
updated